<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="Line" name="rightArm">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="leftArm">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="body">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>200</y>
      <width>16</width>
      <height>121</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="leftLeg">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>71</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLabel" name="rightLeg">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>61</width>
      <height>171</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string>\</string>
    </property>
   </widget>
   <widget class="QLabel" name="head">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>90</y>
      <width>61</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>100</pointsize>
     </font>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="text">
     <string>o</string>
    </property>
   </widget>
   <widget class="Line" name="gallow_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>16</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="gallow">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>90</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="ground">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>460</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="gallow_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>20</width>
      <height>391</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="word">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>120</y>
      <width>191</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
     </font>
    </property>
    <property name="text">
     <string>-------</string>
    </property>
   </widget>
   <widget class="QLabel" name="previous">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>190</y>
      <width>211</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Previous guess:</string>
    </property>
   </widget>
   <widget class="QLabel" name="guess">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>350</y>
      <width>301</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Guess a letter or a word:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="guessBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>440</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="guessButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Guess</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
